pyproject.toml

```diff
@@ -45,7 +45,7 @@ pyright = "^1.1.298"
 pytest-cov = "^4.0.0"
 isort = "^5.11.4"
 pytest-rerunfailures = "^11.1"
-starknet-devnet = "0.5.0a2"
+starknet-devnet = {git = "https://github.com/Shard-Labs/starknet-devnet", rev = "75188cc"}
```

Comment:
Let's add an issue to restore it to pypi if this is merged before devnet releases.